Userland/Libraries/LibJS/Console.cpp

```diff
@@ -526,7 +531,7 @@ ThrowCompletionOr<Value> ConsoleClient::logger(Console::LogLevel log_level, Mark
 ThrowCompletionOr<MarkedVector<Value>> ConsoleClient::formatter(MarkedVector<Value> const& args)
 {
     auto& vm = m_console.vm();
```

Comment:
This doesn't need its own getter, just swap the order and use `realm.vm()`. Generally it's good to minimize the sources of realms, VMs, global objects, and interpreters everywhere.

Reply:
Good to know!